src/users/users.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { User } from "./user.entity";
+import { Repository } from "typeorm";
+import { UserInterface } from "./interfaces/user.interface";
+
+@Injectable()
+export class UsersService {
+    constructor( 
+        @InjectRepository(User)
+        private readonly userRepository: Repository<User>
+    ) {}
+
+    async createUser(username: string, password: string) {
```

Comment:
why do we need all these operations here if we use only one in the controller?
remove these operations or use all of them in controller